src/main/java/org/apache/commons/scxml2/env/jexl/JexlEvaluator.java

```diff
@@ -74,11 +74,21 @@ public Evaluator getEvaluator(final SCXML document) {
     /** The internal JexlEngine instance to use. */
     private transient volatile JexlEngine jexlEngine;
 
+    /** Optional: saves user defined packages, which JEXL should allow for evaluation */
+    private String[] customAllowedClasses;
+
     /** Constructor. */
     public JexlEvaluator() {
         jexlEngine = getJexlEngine();
     }
 
+    /** Constructor with further allowed classes or packages. Use an asterix for all classes in a package */
+    public JexlEvaluator(String... customAllowedClasses) {
```

Comment:
I think we should use the builder pattern instead of adding public constructors.

Reply:
Did some work and added a builder-class which now provides the different options. Moreover it is used in 2 test cases.